docs/index.rst

```diff
@@ -6,15 +6,18 @@ Solidity
     :alt: Solidity logo
     :align: center
 
-Solidity is a contract-oriented, high-level language for implementing smart contracts.
-It was influenced by C++, Python and JavaScript
-and is designed to target the Ethereum Virtual Machine (EVM).
+Solidity is an object-oriented, high-level language for implementing smart
+contracts. Smart contracts are programs which govern the behaviour of accounts
+within the Ethereum state using practically Turing complete computation.
```

Comment:
I would also remove `using practically Turing complete computation`. I think the term "Turing complete" deserves mention in the following two cases:
 - something is not Turing complete
 - something is Turing complete but surprising so

Reply:
@chriseth I agree, also 'practically turing complete' bothered me anyway, what does that mean? :/